differential-expression/README.md

```diff
@@ -1,25 +1,162 @@
+To use GenePattern, you have to create an account.
+Here is [their guide](http://software.broadinstitute.org/cancer/software/genepattern/quick-start) we recommend you follow to get started.
+For use with GenePattern, data from refine.bio needs to be converted to GenePattern
+[formats](http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide).
+If you would like to do differential expression but would prefer using GenePattern, follow the instructions below.
+
+### Preparing your refine.bio data for GenePattern
+
+In order to complete your differential expression analysis using [GenePattern](https://cloud.genepattern.org/gp/pages/login.jsf), you will need to have:  
+ 1) a GCT file    
+ 2) a CLS file  
+
+You can follow the steps below to create these files from your refine.bio data.
+
+#### Create a GCT file
+
+Convert a gene expression tab separated values (TSV) file provided into a 'gene cluster text' (GCT) file for use in GenePattern.
+In order to create a GCT formatted file from a TSV refine.bio data file, download the
+[`create_gct_file.R` script](https://github.com/AlexsLemonade/refinebio-examples/blob/master/scripts/create_gct_file.R).
+To use this script you will need to open [`Terminal`](https://macpaw.com/how-to/use-terminal-on-mac)(for Mac) or [`Command Prompt`](https://www.lifewire.com/command-prompt-2625840)(for Windows).
+You will need to reference the script like the examples below, followed by `--file` argument with the name of the refine.bio dataset TSV file in your current directory that you would like to convert.
+Note: This script requires `optparse` library. If `optparse` library is not installed, this script will install it for you.
+
+**Arguments:**     
+`--file`: name of the file in your current directory that you would like to convert.      
+`--output`: name for the output file, the ".gct" suffix will be added if you do not add it yourself (*optional*).  
+`--rewrite`: file of the same name as the output will be rewritten (*optional*).     
+
+#### Examples of usage in command line:
+
+Below is the basic template for usage of this script.
+The following examples will give you an idea of how it works.
+
+```
+Rscript scripts/create_gct_file.R \
+  --file <PATH TO REFINE.BIO EXPRESSION TSV>
+```
+To get an idea of how this script and its arguments work, you can run the following examples in order.
+
+**Go the correct directory**  
+Depending on where you have put the `refinebio-examples` directory on your computer, you will have to change this path in the code chunk below.
+Be sure to either have the script and input file in your current working directory, or put type out the full directory path for the script and/or input file. eg. `/users/Bob/Desktop/scripts/create_gct_file.R`
+For more guidance on how to navigate directories, we recommend [this tutorial](https://swcarpentry.github.io/shell-novice/02-filedir/index.html).
+
+```bash
```

Comment:
I would either take this chunk or add more context. For example, you'd likely want to point out that this assumes that `refinebio-examples` has been copied to your Desktop.

Reply:
I will take it out, because the information given in the paragraph *should* help them accomplish this step, but if they blindly put in the code chunk and didn't read the paragraph, they may be confused.